services-camel/src/test/java/edu/unc/lib/dl/services/camel/BinaryMetadataProcessorTest.java

```diff
@@ -141,6 +147,7 @@ public void noLocalBinaryTest() throws Exception {
         verify(message).setHeader(CdrBinaryChecksum, checksum);
         verify(message).setHeader(CdrBinaryMimeType, mimetype);
         verify(message, never()).setHeader(eq(CdrBinaryPath), anyString());
+        verify(message, never()).setHeader(eq(CdrBinaryId), anyString());
```

Comment:
If this class can never set CdrBinaryId, then you don't really need to test for it